moveit_ros/moveit_servo/include/moveit_servo/servo.hpp

```diff
@@ -134,10 +134,8 @@ class Servo
 private:
   /**
    * \brief Convert a give twist command to planning frame
-   * This implementation does not differentiate between body twist and spatial twist,
-   * i.e if the command is as seen from a stationary or non-stationary frame.
-   * The result of this transformation will only be accurate if the command frame is a stationary frame.
-   * See issue: https://github.com/ros-planning/moveit2/issues/2150
+   * The reference frame specified by `frame_id` should be a stationary frame.
+   * The command may be expressed in the space frame basis or the body frame basis.
```

Comment:
>    * This implementation does not differentiate between body twist and spatial twist,
>    * i.e if the command is as seen from a stationary or non-stationary frame.

This part is no longer true since command can now be given as in body frame or space frame.

>    * The result of this transformation will only be accurate if the command frame is a stationary frame.

can keep this part.

Reply:
Well, don't these 2 sentences contradict each other?

>   * The reference frame specified by `frame_id` should be a stationary frame.
>   * The command may be expressed in the space frame basis or the body frame basis.

Reply:
I agree this part might be a bit confusing.
The question is, in the context of twist command, does `frame_id` represent the reference frame or the basis in which the command is expressed ?
I was thinking of it as the reference frame

Eg: B moving w.r.t to A, --> command reference frame A, but expressed in B. 
Should `frame_id` be A or B?

Reply:
I think if the user is commanding something in the end-effector frame, then `command.header.frame_id` should be `end_effector` (whatever the EE frame is named).

If they're commanding something in the planning frame then `command.header.frame_id` should be `base_link`.

I think it's safe to assume for Servo that the user always wants to move the end effector. They won't move any other links. If they wanted to move a different link, they could set up a new planning group to do that.

For example, if `command.header.frame_id` is "elbow" .... that means the basis for the command is the elbow frame but it is still the end effector that will move in that direction.